stl/debugger/STL.natvis

```diff
@@ -400,6 +400,14 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
       </Expand>
   </Type>
 
+  <!-- VS 2019 16.3 -->
+  <Type Name="std::_Ref_count_obj2&lt;*&gt;">
+      <DisplayString>make_shared</DisplayString>
+      <Expand>
+          <Item Condition="_Uses != 0" Name="[original ptr]">($T1 *) &amp;_Storage</Item>

```

Comment:
Can we add a fixed string to a `_Uses == 0` condition?

Reply:
Why would we need to show a child item for an empty shared_ptr? (weak_ptrs will display themselves as expired, IIRC)